tests/std/tests/VSO_0000000_instantiate_containers/test.compile.pass.cpp

```diff
@@ -623,6 +685,12 @@ void string_test_impl(const CharType* data) {
 
     erase_if_test(value);
     erase_test(value);
+
+#ifndef _M_CEE // TRANSITION, VSO-1659496
+    validating_converter arr[1]{};
+    another.append(+arr, +arr);
+    another.replace(cbegin(value), cbegin(value), +arr, +arr);

```

Comment:
Although this test is compile-only, this call would be bogus at runtime - the first pair of iterators are pointing into the wrong string. (Also, this says `cbegin` twice which is weird.)

I don't think we need to exactly imitate `value.replace(cbegin(value), cend(value), begin(another), end(another));` above, but we should at least say `cbegin(another), cend(another)` for the first pair of iterators here.